tests/test_problem_generator.py

```diff
@@ -0,0 +1,71 @@
+import unittest
+
+from ai_graph_color import problem_generator
+
+
+class TestProblemGenerator(unittest.TestCase):
+    def test_create_file(self):
+        """
+        Tests create file
+        """
+        file_names = ['test.json', '']
+        for file_name in file_names:
+            file_path = problem_generator.create_file(file_name)
+            self.assertEqual(type(''), type(file_path))
+            self.assertEqual('problems/{}'.format(file_name), file_path)
+
+    def test_read_and_write_graph_to_file(self):
+        """
+        Tests write graph to file
+        """
+        num_verts = [0, 5, 100]
+        for index, num_vert in enumerate(num_verts):
+            problem_generator.write_graph_to_file(
+                'test{}.json'.format(index),
+                problem_generator.generate_graph(num_vert)
+            )
+
+            # This will need to change once generate graph is done
```

Comment:
Maybe just test that the thing you read from the file is the same as what you wrote to the file?